index.html

```diff
@@ -399,12 +400,80 @@ <h1>Documentation</h1>
     </ul>
   </div>
 
+        <!-- mainline releases v4.19.0.1 -->
+        <div class="wrapper">
+          <!--<div class="break"></div>insert this code when a new update comes out to separate a new version from the current.-->
+          <div class="smaller-margin" id="mr-63">
+            <h2 class="text-middle">Mainline releases</h2>
+            <!-- h2 tag needs to be moved to the newest version -->
+          <h4><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.17.0.3...rel_4.19.0.1">v4.19.0.1</a></h4>
+        </div>
+  
+        <div class="dimmed-text">
+          <p>Release date: 2023-01-27</p>
```

Comment:
Date update? tag was placed on 31th of January

Reply:
Could we use in this place tag placed date or release date?

Reply:
Based on the previous version's dates in this file, it should be the date of release publication on github.io page (so today 2023-02-02)